test/functional/api/v1/deletions_controller_test.rb

```diff
@@ -100,7 +100,8 @@ class Api::V1::DeletionsControllerTest < ActionController::TestCase
       context "ON DELETE to create for someone else's gem" do
         setup do
           @other_user = create(:user)
-          @request.env["HTTP_AUTHORIZATION"] = @other_user.api_key
+          create(:ownership, user: @other_user, rubygem: @rubygem)
+          @ownership.destroy
```

Comment:
I don't like that `@ownership` is destroyed here. It looks confusing.
I think test would be clearer if we created a different gem (different from `@rubygem`) for `@other_user` and `@user` tried to delete that.